.env.example

```diff
@@ -1 +1,3 @@
 NEXT_PUBLIC_BASE_URL=https://dev-next-media.g.kuroco.app
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET="GCpO5jjFi654BYL9nfoUJos6PGI52dnViR5ZCzdIMd0="
```

Comment:
What is this for?

Reply:
Carried over from initial NextAuth implementation. 
Was redundant. Resolved now.